Assignment2Conditionals/src/assignment2conditionals/Assignment2Conditionals.java

```diff
@@ -0,0 +1,107 @@
+package assignment2conditionals;
+
+import java.util.Scanner;
+
+public class Assignment2Conditionals 
+{
+
+    public static void main(String[] args) 
+    {
+     Scanner scan = new Scanner (System.in);
+           
+     System.out.println("\t*************************************\n"
+                      + "\t*       Steve's Calculator          *\n"
+                      + "\t*************************************");
+          
+     System.out.println(" 1: Math\n 2: Subtraction\n 3: Multiplication"
```

Comment:
For the sake of readability I would recommend more liberal use of the `+` **continuation** symbol like this
```Java
System.out.println( "1: Math\n"
		          + "2: Subtraction\n"
			      + "3: Multiplication\n"
                  + "4: Devision\n"
			      + "5: Word Counter\n"
		          + "6: Exit Program\n");